### DISCUSSION TITLE
Have the player indicator overlay stack

### DISCUSSION THREAD

**Discussion by Stickaplex** [runelite/runelite#15029] [Ideas]
Currently if multiple clannies/friends are standing on the same square, the names go on top of each other making it impossible to read. Would look much better if they went over each other similiar to how ground items are.

**Comment by Nightfirecat** [comment]
After thinking about this a little, this is harder than it seems. Currently, that overlay draws names based on each player's height, which is often slightly different due to their animation or equipped items, so that would have to first be changed before iterating over players sharing a single tile.

**Comment by deathbeam** [comment]
jumpy player names is best part of the plugin :( i would rather have that than stacking names tbh

**Comment by alexanderhenne** [comment]
An example of how it can look: 
https://twitter.com/_lluvata/status/1117161020911435777

![image](https://user-images.githubusercontent.com/8920674/56086403-ee511380-5e56-11e9-87ee-a961d0143306.png)

**Comment by zeruth** [comment]
Even if you stacked them per tile, groups of players would still be unintelligible. Good maybe to know if a clan is stacking a spot I suppose but not very useful elsewhere, and would make places like the GE even more annoying.

**Comment by Nightfirecat** [comment]
Without doing a lot of work, I don't think there's an easy (or a good) solution to this.
